vanagon.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |gem|
   gem.email    = 'info@puppet.com'
   gem.homepage = 'http://github.com/puppetlabs/vanagon'
   gem.specification_version = 3
-  gem.required_ruby_version = '~> 2.1'
+  gem.required_ruby_version = '~> 2.4'
```

Comment:
Do we actually want to require 2.4 or above? I'm not sure about this bit

Reply:
Likely minimum should be whatever we have available on vmpooler machines.

Reply:
At this point, packaging requires at least 2.3 (via transitive dependencies), so probably don't want to go below that..